package.json

```diff
@@ -6,7 +6,12 @@
   "scripts": {
     "test": "npx hardhat --network localhost test",
     "lint": "eslint $(git ls-files '*.ts')",
-    "format": "prettier-eslint --list-different --write $(git ls-files '*.ts')"
+    "format": "prettier-eslint --list-different --write $(git ls-files '*.ts')",
+    "format-check": "prettier-eslint $(git ls-files '*.ts') --list-different",
+    "check-executable": "./util_scripts/check_executable.sh",
+    "link-plugin": "./util_scripts/link_master_plugin.sh",
+    "check-it-only-file": "grep -rq 'it\\.only(' $(git ls-files '*.ts') && echo 'Error: There are files with it.only(...)' && exit 2 || true",
```

Comment:
Why `|| true`?

Reply:
As we don't want the script to fail when no occurrences are found, It ensures ensure that the overall exit code of the script is 0

Reply:
Ah, that's because grep returns exit code 1 if it doesn't find a match? Btw, I just realized the script could be even better: we could remove the `-q` (quiet) flag so that it prints the actual matches (for easier fixing).

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-example/pull/122/commits/8cff0f3e343319164ad6d1244b4b65e2d49f99c6